components/autoComplete/autoCompleteItem.js

```diff
@@ -1,5 +1,6 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
+import omit from 'lodash.omit';
```

Comment:
according to this comment https://github.com/Travix-International/travix-ui-kit/pull/221#discussion_r149354939 better to use `lodash/omit`